Dockerfile

```diff
@@ -1,11 +1,33 @@
-FROM python:2.7 
+# getting the base Image
+FROM python:3.6
```

Comment:
Did you check what's the size of the resultant image? We may need to tinker with the base image to get the best results.

Reply:
Yes the image size was 1.2GB with python 3.6 having 916MB. I just change the image version to keep update to date with the current documentation.

Reply:
Can you try with `3.6-slim-buster`, that is the image I use, and as far as I know that's the image with the smallest footprint

Reply:
ok sure previously actually I used 3.6-slim and 3.6-alpine but it gave me error.

Reply:
Thanks for that it reduced the size of the image to 600MB. I actually didn't use it first because the above-specified versions gave me an error that's why I keep as it is.